src/manage.c

```diff
@@ -298,6 +299,12 @@ static acq_status_t * choose_acq_sat(void)
   gps_time_t t = get_current_time();
 
   for (u32 i=0; i<PLATFORM_SIGNAL_COUNT; i++) {
+    if (CODE_GPS_L2CM == acq_status[i].sid.code) {
```

Comment:
I added the new API function code_requires_direct_acq() into libswiftnav. Please check https://github.com/swift-nav/libswiftnav-private/pull/6: 
https://github.com/adel-mamin/libswiftnav-private/blob/9ce04464eb17c7a76e7882b83ac4b5e20a7bf52d/src/signal.c#L280